File: 2 - Operators/Solution.cs
using System;

class Solution
{
    static void Main(String[] args)
    {
        var mealCost = double.Parse(Console.ReadLine());
        var tipPercent = int.Parse(Console.ReadLine());
        var taxPercent = int.Parse(Console.ReadLine());

        var tip = tipPercent * mealCost / 100;
        var tax = taxPercent * mealCost / 100;

        var totalCost = Math.Round(tip + tax + mealCost);
        Console.WriteLine($"The total meal cost is {totalCost} dollars.");
    }
}

File: 2 - Operators/Solution.go
package main

import (
	"fmt"
	"math"
)

func round(num float64) float64 {
	if num < 0 {
		return math.Ceil(num - 0.5)
	}
	return math.Floor(num + 0.5)
}

func main() {
	var mealCost float64
	var tipPercent int
	var taxPercent int

	fmt.Scan(&mealCost)
	fmt.Scan(&tipPercent)
	fmt.Scan(&taxPercent)

	tip := mealCost * (float64(tipPercent) / 100)
	tax := mealCost * (float64(taxPercent) / 100)
	totalCost := mealCost + tip + tax
	var t = int(round(totalCost))
	fmt.Printf("The total meal cost is %d dollars.", t)

}
